src/mcp/prompts/core/init.ts

```diff
@@ -18,14 +19,38 @@ export const init = prompt(
     },
   },
   async ({ prompt }, { config, projectId, accountEmail }) => {
+    const shortCircuitUrls: Record<string, string> = {
+      "ai-logic": "firebase://guides/init/ai",
+      backend: "firebase://guides/init/backend",
+    };
+    const shortCircuitUrl = prompt ? shortCircuitUrls[prompt] : undefined;
+
     const platform = await getPlatformFromFolder(config.projectDir);
 
     return [
       {
         role: "user" as const,
         content: {
           type: "text",
-          text: `
+          text: shortCircuitUrl
+            ? makeShortCircuitPrompt(shortCircuitUrl)
+            : makeDefaultPrompt(platform, projectId, accountEmail, config),
+        },
+      },
+    ];
+  },
+);
+
```

Comment:
I was imagining we would literally just programmatically load the content of the short-circuited-to guide (i.e. import `init_ai` and inline it in place of the default prompt) rather than making the model do a tool call to retrieve.

Reply:
Let me try that out